activestorage/__init__.py

```diff
@@ -0,0 +1 @@
+from .active import Active
```

Comment:
I don't think this is needed, unless we move things done in modules here in the init

Reply:
How else can we import `Active` in user code?

Reply:
they'll have to import it at package-level from the `activestorage` package if they want to, if it's internal then all module imports happen from module